traitsui/extras/_demo_legacy.py

```diff
@@ -0,0 +1,1187 @@
+# -------------------------------------------------------------------------
+#
+#  Copyright (c) 2005, Enthought, Inc.
+#  All rights reserved.
+#
+#  This software is provided without warranty under the terms of the BSD
+#  license included in LICENSE.txt and may be redistributed only
+#  under the conditions described in the aforementioned license.  The license
+#  is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+#  Thanks for using Enthought open source!
+#
+#  Author: David C. Morrill
+#  Date:   09/15/2005
+#
+# -------------------------------------------------------------------------
+
+""" A Traits UI demo that borrows heavily from the design of the wxPython demo.
+
+This copy of the demo application will only be maintained until ``ets-demo``
+is released and distributed. This allows other ETS packages to continue to be
+able to import the demo application until ``ets-demo`` is released.
+"""
```

Comment:
The rest of this file was identical to the removed `traitsui/extras/demo.py`, except for this paragraph.
I should open a new issue for removing this legacy file eventually. Will do that now.